lib/aff4_objects/security.py

```diff
@@ -151,10 +151,11 @@ class ApprovalWithApproversAndReason(Approval):
 
   class SchemaCls(Approval.SchemaCls):
     """The Schema for the ClientAccessApproval class."""
+
     LIFETIME = aff4.Attribute(
         "aff4:approval/lifetime", rdfvalue.RDFInteger,
-        "The number of microseconds an approval is valid for.",
-        default=4 * 7 * 24 * 60 * 60 * 1000000)  # 4 weeks
+        "The number of seconds an approval is valid for.",
```

Comment:
This should remain microseconds because all of the timestamps and comparisons are in microseconds.  This is probably where it should be getting the value out of the config.